docs/Makefile

```diff
@@ -0,0 +1,192 @@
+# Makefile for Sphinx documentation
```

Comment:
@intellectronica They're mostly the defaults. I think it's probably easiest just to let the software build what it wants to.